src/uct/ib/rc/base/rc_iface.c

```diff
@@ -184,7 +184,7 @@ ucs_status_t uct_rc_iface_query(uct_rc_iface_t *iface,
     iface_attr->cap.get.max_iov   = uct_ib_iface_get_max_iov(&iface->super);
 
     /* AM */
-    iface_attr->cap.am.max_short  = max_inline - tag_min_hdr;
+    iface_attr->cap.am.max_short  = UCT_IB_MAX_INLINE_VAL(max_inline, tag_min_hdr);
```

Comment:
can we reuse us_min/ucs_max macros?

Reply:
can be, but due to unsigned datatypes construct will looks like:
```val = ucs_max(max_inline, sub) - sub```
current implementation is shorter, but I can change it :)